setup.py

```diff
@@ -46,6 +44,8 @@
     name=NAME,
     version=VERSION,
     description=DESCRIPTION,
+    long_description=long_description,
+    long_description_content_type="text/markdown",
```

Comment:
Nit: change double quotes to single quotes for consistency.

Reply:
Done